Cubical/Categories/Equivalence/Base.agda

```diff
@@ -24,6 +24,69 @@ record WeakInverse {C : Category ℓC ℓC'} {D : Category ℓD ℓD'}
     η : 𝟙⟨ C ⟩ ≅ᶜ invFunc ∘F func
     ε : func ∘F invFunc ≅ᶜ 𝟙⟨ D ⟩
 
+-- Composition of weak inverses is a weak inverse
```

Comment:
unsure if here or `Properties` is better

Reply:
Without looking at it: We usually do not split up things enough in the library, so most of the time the good idea is to have more files. So it might also sense to have something on 'composition' or 'weak equivalences'.